nano/templates/sleeper.tmpl

```diff
@@ -0,0 +1,201 @@
+				<div class="statusValue">{{:helper.round(data.occupant.bruteLoss)}}</div>
+			</div>
+		
+			<div class="line">
+				<div class="statusLabel">=&gt; Resp. Damage:</div>
+				{{:helper.displayBar(data.occupant.oxyLoss, 0, data.occupant.maxHealth, 'bad')}}
+				<div class="statusValue">{{:helper.round(data.occupant.oxyLoss)}}</div>
+			</div>
+		
+			<div class="line">
+				<div class="statusLabel">=&gt; Toxin Damage:</div>
+				{{:helper.displayBar(data.occupant.toxLoss, 0, data.occupant.maxHealth, 'bad')}}
+				<div class="statusValue">{{:helper.round(data.occupant.toxLoss)}}</div>
+			</div>
+		
+			<div class="line">
+				<div class="statusLabel">=&gt; Burn Severity:</div>
+				{{:helper.displayBar(data.occupant.fireLoss, 0, data.occupant.maxHealth, 'bad')}}
+				<div class="statusValue">{{:helper.round(data.occupant.fireLoss)}}</div>
+			</div>
+			
+			<br>
+			<div class="line">
+			<!--
+				<div class="statusLabel">Patient Temperature:</div>
+				{{if data.occupant.temperatureSuitability == -3}}
+				{{:helper.displayBar(data.occupant.bodyTemperature, 0, data.occupant.maxTemp, 'cold3')}}
+				<div class="statusValue bad">{{:helper.round(data.occupant.btCelsius)}}C, {{:helper.round(data.occupant.btFaren)}}F</div>
+				{{else data.occupant.temperatureSuitability == -2}}
+				{{:helper.displayBar(data.occupant.bodyTemperature, 0, data.occupant.maxTemp, 'cold2')}}
+				<div class="statusValue bad">{{:helper.round(data.occupant.btCelsius)}}C, {{:helper.round(data.occupant.btFaren)}}F</div>
+				{{else data.occupant.temperatureSuitability == -1}}
+				{{:helper.displayBar(data.occupant.bodyTemperature, 0, data.occupant.maxTemp, 'cold1')}}
+				<div class="statusValue">{{:helper.round(data.occupant.btCelsius)}}C, {{:helper.round(data.occupant.btFaren)}}F</div>
+				{{else data.occupant.temperatureSuitability == 0}}
+				{{:helper.displayBar(data.occupant.bodyTemperature, 0, data.occupant.maxTemp, 'good')}}
+				<div class="statusValue">{{:helper.round(data.occupant.btCelsius)}}C, {{:helper.round(data.occupant.btFaren)}}F</div>
+				{{else data.occupant.temperatureSuitability == 1}}
+				{{:helper.displayBar(data.occupant.bodyTemperature, 0, data.occupant.maxTemp, 'hot1')}}
+				<div class="statusValue">{{:helper.round(data.occupant.btCelsius)}}C, {{:helper.round(data.occupant.btFaren)}}F</div>
+				{{else data.occupant.temperatureSuitability == 2}}
+				{{:helper.displayBar(data.occupant.bodyTemperature, 0, data.occupant.maxTemp, 'hot2')}}
+				<div class="statusValue bad">{{:helper.round(data.occupant.btCelsius)}}C, {{:helper.round(data.occupant.btFaren)}}F</div>
+				{{else data.occupant.temperatureSuitability == 3}}
+				{{:helper.displayBar(data.occupant.bodyTemperature, 0, data.occupant.maxTemp, 'hot3')}}
+				<div class="statusValue bad">{{:helper.round(data.occupant.btCelsius)}}C, {{:helper.round(data.occupant.btFaren)}}F</div>
+				{{/if}}
+				If you can get the changing temperature bars to work right, congratulations
```

Comment:
I wanted to do something cool here, where the status bar would be darker shades of blue the colder the patient got, and vice-versa for heat, but it wound up always being a light blue when not at a healthy temperature. Any ideas for a fix?